axelrod/tests/strategies/test_finite_state_machines.py

```diff
@@ -780,3 +784,54 @@ def test_strategy(self):
         # finished 0, 1, 2, 3, 4, 5, 6, 8, 10, 11, 12, 13, 14, 15
         state_and_actions = to_state_eleven + [(11, C), (14, D)] + [(13, C)] * 3
         self.transitions_test(state_and_actions)
+
+
+class TestTF1(TestFSMPlayer):
+    name = "TF1"
+    player = axelrod.TF1
+    expected_classifier = {
+        'memory_depth': 16,
+        'stochastic': False,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        self.first_play_test(C)
```

Comment:
As a regression test could you include a 5 turn play against alternator? (Same for the other 2)